.rubocop.yml

```diff
@@ -120,6 +120,87 @@ Layout/SpaceInsideParens:
   Exclude:
     - 'db/migrate/20130705092519_create_rails_admin_histories_table.rb'
 
+Style/FrozenStringLiteralComment:
+  Exclude:
+    - 'Gemfile'
+    - 'Rakefile'
+    - 'bin/bundle'
+    - 'bin/rails'
+    - 'bin/rake'
+    - 'config.ru'
+    - 'db/migrate/20111123174941_devise_create_users.rb'
```

Comment:
Should we maybe disable this rule? Having to add a line here every time we add migration seems like it'd get tiring.

Reply:
For new file, magic comment should be added. I just don't want to modify old migration files.